test/schedule/block_test.exs

```diff
@@ -80,6 +80,31 @@ defmodule Schedule.BlockTest do
            pieces: [@piece]
          )
 
+  @as_directed %AsDirected{
+    id: "ad1",
+    kind: :wap,
```

Comment:
😂 um

```suggestion
    kind: :wad,
```
Interesting that this didn't lead to failed tests?

Reply:
After looking at it `next_revenue_trip`, I'm not concerned there isn't a failed test, as that function wasn't responsible for checking it. But why didn't elixir's typing catch this??

Reply:
oh wow thanks for catching this @hannahpurcell !